src/main/java/org/truffleruby/parser/Translator.java

```diff
@@ -60,6 +60,7 @@
         m.put("$INPUT_RECORD_SEPARATOR", "$/");
         m.put("$>", "$stdout");
         m.put("$PROGRAM_NAME", "$0");
+        m.put("$CHILD_STATUS", "$?");
```

Comment:
That should be aliased by `lib/mri/English.rb`, but maybe it doesn't work for us yet?